app/src/components/governance/ProposalHeader.vue

```diff
@@ -0,0 +1,293 @@
+<template>
+  <header class="header">
+    <div class="header-top-column">
+      <div class="header-top-row">
+        <div>
+          <BackButton />
+        </div>
+
+        <div class="inner-status">
+          <Status :status="status" />
+        </div>
+
+        <div class="buttons">
+          <button
```

Comment:
refactor the copy button? the same logic exists for address copy

Reply:
what do you mean? Refactor how? It is already working, I copy pasted the logic from the Address component, exactly

Reply:
i think fabo is saying that we already have a click-to-copy component (address) and so it would be better to generalize it so it could be used for any strings / links to avoid duplicating the code here. 

IMO should be another PR because this one is already huge and it's a nice-to-have refactor but not critical.

Reply:
Yes, let's get this one in and then we can do all these small enhances. It would be a good idea to open an issue with all of them so we don't forget and then work on them in different mini-PRs

Reply:
OK, I opened the issue, #4896